core/arch/arm/kernel/thread_a64.S

```diff
@@ -986,6 +987,16 @@ END_FUNC el0_sync_abort
 	msr	spsel, #0
 	mov	sp, x1
 
+#ifdef CFG_CORE_WORKAROUND_NSITR_CACHE_PRIME
+	/*
+	 * Prevent leaking information about which entries has been used in
+	 * cache. We're relying on the dispatcher in TF-A to take care of
+	 * the BTB.
+	 */
+	mov	x0, #DCACHE_OP_CLEAN_INV
+	bl	dcache_op_louis
+	ic	iallu
```

Comment:
likewise .. Invalidating by range only sec ram.

Reply:
Same as above, hard to predict narrow ranges.